Components/Images/images.css

```diff
@@ -0,0 +1,18 @@
+.image-responsive{
+    width: 100%;
+    max-width:50vw;

```

Comment:
how max-width will work on the image? 
as we have already defined width as 100%

I didn't get the use case of max-width and width, 
max-width can be given on your image wrapper(image-responsive-wrapper) div

Reply:
max-width will restrict the image to 50vw and 100% will give it cover whole 50vw .